### DISCUSSION TITLE
Are you saving data of source data sources in the intermediate data store ?

### DISCUSSION THREAD

**Discussion by anitha-gadde** [open-metadata/OpenMetadata#7975] [Q&A] [answered]
Are you saving data of source data sources in the intermediate data store ?  if yes where are you storing source tables actual data?

**Comment by pmbrull** [comment, marked as answer]
Hi, we are not saving any source data, only metadata. As part of the metadata we can optionally get sample rows (only if you enable the extraction).

All of this will be stored in the underlying db https://docs.open-metadata.org/main-concepts/high-level-design
